Makefile

```diff
@@ -111,21 +111,30 @@ lint-lua: FORCE
 lint-sh: FORCE
 	@scripts/lint-sh.sh
 
-GLUON_DEFAULT_PACKAGES := hostapd-mini
+GLUON_DEFAULT_PACKAGES :=
+GLUON_FEATURESET_STANDARD_PACKAGES := gluon-featureset-standard
+GLUON_FEATURESET_TINY_PACKAGES := gluon-featureset-tiny
 
-GLUON_FEATURE_PACKAGES := $(shell scripts/features.sh '$(GLUON_FEATURES)' || echo '__ERROR__')
-ifneq ($(filter __ERROR__,$(GLUON_FEATURE_PACKAGES)),)
-$(error Error while evaluating GLUON_FEATURES)
-endif
+GLUON_FEATURESET_STANDARD_FEATURE_PACKAGES := 
+GLUON_FEATURESET_TINY_FEATURE_PACKAGES := 
 
+define feature_packages
+  $(1) := $(shell scripts/features.sh '$(2)' || echo '__ERROR__')
+endef
+$(eval $(call feature_packages, GLUON_FEATURESET_STANDARD_FEATURE_PACKAGES, $(GLUON_FEATURES) $(GLUON_FEATURESET_STANDARD_FEATURES)))
+$(eval $(call feature_packages, GLUON_FEATURESET_TINY_FEATURE_PACKAGES, $(GLUON_FEATURES) $(GLUON_FEATURESET_TINY_FEATURES)))
```

Comment:
I think it would make sense to reuse `merge_packages` here (renamed to `merge_lists`), so the featureset featurelists can contain negative entries.